s2n.mk

```diff
@@ -142,6 +142,11 @@ ifeq ($(S2N_UNSAFE_FUZZING_MODE),1)
     # Create a marker file so that later invocations of make can pick the right COV_TOOL by default
     $(shell touch "${LLVM_GCOV_MARKER_FILE}")
 
+	# Filter out the visibility settings if we are fuzzing
+	CFLAGS:=$(filter-out -fvisibility=hidden,$(CFLAGS))
```

Comment:
Nit: Indentation is off

Reply:
turns out this is what was breaking my fuzz test changes.